src/forms/SelectInput.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module SelectInput
+ */
+class SelectInput extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value || props.options[0].value
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onChange(e) {
+		this.setState({ value: e.target.value });
+		if (this.props.onChange) {
+			this.props.onChange(e);
+		}
+	}
+
+	render() {
+		const {
+			children,
+			className,
+			labelClassName,
+			label,
+			options,
+			name,
+			errors,
+			required,
+			onChange, // eslint-disable-line no-unused-vars
+			value, // eslint-disable-line no-unused-vars
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			{ 'field--error': errors && errors.length },
+			className
+		);
+
+		const labelClassNames = cx(
+			{ required },
+			labelClassName
+		);
+
+		return (
+			<div>
+				<label className={labelClassNames} htmlFor={other.id}>
+					{label}
+				</label>
+
+				<select
+					name={name}
+					required={required}
+					className={classNames}
+					onChange={this.onChange}
+					value={this.state.value}
+					{...other}>
```

Comment:
putting the closing `>` at the same indentation level as the opening `<` will show that the `{` on the line below needs an extra indentation level.